doc/source/_toc.yml

```diff
@@ -96,7 +96,7 @@ parts:
           - file: data/getting-started
           - file: data/key-concepts
           - file: data/user-guide
-          - file: data/examples
+          - file: data/data_examples
```

Comment:
```suggestion
          - file: data/data_examples
            title: Examples
```

Reply:
Are you open to keeping the name the same?

Reply:
The name isn't changed in this PR. It's still being pulled from the title of data_examples.rst

Reply:
I think you can set it with that `title` line, no?

Reply:
If not provided, it will default to the title in the rst file